beacon_node/beacon_chain/src/attestation_verification.rs

```diff
@@ -539,6 +539,19 @@ impl<'a, T: BeaconChainTypes> IndexedAggregatedAttestation<'a, T> {
             Err(e) => return Err(SignatureNotChecked(&signed_aggregate.message.aggregate, e)),
         };
 
+        let relevant_epoch = signed_aggregate.message.epoch().saturating_sub(1u64);
+        let aggregator = signed_aggregate.message.aggregator_index as usize;
+
+        chain
+            .effective_balances_cache
+            .load_epoch(chain, relevant_epoch)
```

Comment:
This load, then retrieve strategy is to get around the lock issues?

Reply:
I don't see any lock issues here? Most of the time the cache should be full anyway. This just ensures it's primed.